content/en/security/application_security/threats/threat-intelligence.md

```diff
@@ -42,15 +42,11 @@ To query for all traces containing threat intelligence from any source, use the
 
 ## Bring your own threat intelligence
 
-{{< callout url="https://forms.gle/JV8VLH1ZTzmUnK5F7" d-toggle="modal" d_target="#signupModal" custom_class="sign-up-trigger">}}
-  Bring your own threat intelligence (BYOTI) is in private beta.
-{{< /callout >}} 
-
 ASM supports enriching and searching traces with threat intelligence indicators of compromise stored in Datadog reference tables. [Reference Tables][2] allow you to combine metadata with information already in Datadog.
 
 ### Storing indicators of compromise in reference tables
 
-Threat intelligence is supported in the CSV format and requires 4 columns.
+Threat intelligence is supported in the CSV format and requires the following columns.
```

Comment:
If stating "the following", I would end with a colon `:`, but if using "the below", I would end with a period `.`